### ISSUE TITLE
App shit

### ISSUE THREAD

**Issue by Neelgodly** [AdityaJS12321/Password-Generator#1]
This app shit

**Comment by AdityaJS12321** [comment]
Stfu asshole

**Comment by Neelgodly** [comment]
no u

**Comment by AdityaJS12321** [comment]
Hello. Thank you for getting in touch regarding the app. If you have the time, can you provide some further information here so that we can assist you with the problem that you've had.


Many thanks and looking forward to hearing from you :)

**Comment by Neelgodly** [comment]
yes, the problem is that full shit
like 100% shit
like shit machine set to max shit
like crap shit, poop, nonsense

**Comment by AdityaJS12321** [comment]
In that case you better get your head into it and give it some company. At least the app wont remain hopeless.

**Comment by Neelgodly** [comment]
> In that case you better get your head into it and give it some company. At least the app wont remain hopeless.

no, its hopeless shit. putting diamond into shit wont convert the shit to diamond so keep your shit to yourself

**Comment by Pranavkool** [comment]
> > In that case you better get your head into it and give it some company. At least the app wont remain hopeless.
> 
> no, its hopeless shit. putting diamond into shit wont convert the shit to diamond so keep your shit to yourself

Our aim wasn't to convert shit to diamond. It was to add more shit into it, so that we can distribute more shit around the world

**Comment by Neelgodly** [comment]
> > > In that case you better get your head into it and give it some company. At least the app wont remain hopeless.
> > 
> > 
> > no, its hopeless shit. putting diamond into shit wont convert the shit to diamond so keep your shit to yourself
> 
> Our aim wasn't to convert shit to diamond. It was to add more shit into it, so that we can distribute more shit around the world

well i cant help you with that cause i ain't got no shit in me

**Comment by Pranavkool** [comment]
> well i cant help you with that cause i ain't got no shit in me

I see nobody has had the balls to reveal it to you. I don't want you to be in the dark about this, but in case you missed it, your head is filled with shit.

**Comment by AdityaJS12321** [comment]
> > > > In that case you better get your head into it and give it some company. At least the app wont remain hopeless.
> > > 
> > > 
> > > no, its hopeless shit. putting diamond into shit wont convert the shit to diamond so keep your shit to yourself
> > 
> > 
> > Our aim wasn't to convert shit to diamond. It was to add more shit into it, so that we can distribute more shit around the world
> 
> well i cant help you with that cause i ain't got no shit in me

you're right, you aint got anything that's not shit

**Comment by Neelgodly** [comment]
> you're right, you aint got anything that's not shit

Huh what crap? did you even learn english
You know what? Ditch this shit. I don't want to talk to developers who dont show respect to their users.
Go on, I hope you fail miserably in your mission to turn this world into SHIT

**Comment by AdityaJS12321** [comment]
We most certainly will, you only need to watch your back from now on.

**Comment by Neelgodly** [comment]
woopsie app became shitter than before, so had to reopen issue
PLEASE UNSHIT

**Comment by AdityaJS12321** [comment]
Hmm, we are looking into it. We are sorry for the inconvenience

Enter your credit card information for a refund

**Comment by Neelgodly** [comment]
nah sux. keep the money. you'll need it to unshit

**Comment by AdityaJS12321** [comment]
cope seethe mald

**Comment by Neelgodly** [comment]
App looks to be half unshitty, hence removing this issue. But please prevent crash because that still shit

**Comment by Neelgodly** [comment]
app has been shitted again

**Comment by AdityaJS12321** [comment]
No it's only half shit, half of the fuckery has been juiced

**Comment by Neelgodly** [comment]
> No it's only half shit, half of the fuckery has been juiced

well half shit is still shit so...

**Comment by AdityaJS12321** [comment]
Nah half shit is half shit not full shit like you

**Comment by Neelgodly** [comment]
shit is shit, half or full doesnt matter

> Nah half shit is half shit not full shit like you

**Comment by AdityaJS12321** [comment]
If you really are disappointed with the app, please enter your credit card info and let us help you. You dont need to be in distress because of our app and bring in a bad name. You gotta accept this time cause I have your ip address

**Comment by Neelgodly** [comment]
> If you really are disappointed with the app, please enter your credit card info and let us help you. You dont need to be in distress because of our app and bring in a bad name. You gotta accept this time cause I have your ip address

guess what, i have urs too, lets exchange

**Comment by AdityaJS12321** [comment]
I use vpn 😎

**Comment by Neelgodly** [comment]
> I use vpn 😎

yeah sure bich, you think your free vpns can protect you? Cheap ass fool, can't afford a paid vpn!

**Comment by AdityaJS12321** [comment]
who told ya its paid

**Comment by Neelgodly** [comment]
> who told ya its free

you think i need someone to tell me huh?

**Comment by AdityaJS12321** [comment]
fake
I use Nord

**Comment by Neelgodly** [comment]
The admin team has taken me as a dev and instantly getting to work, I have fixed this issue with PR #7
